Ladybird/AppKit/UI/LadybirdWebView.mm

```diff
@@ -615,6 +615,14 @@ - (void)setWebViewCallbacks
                                   url:url
                           activateTab:Web::HTML::ActivateTab::Yes];
     };
+
+    m_web_view_bridge->on_theme_color_change = [self](auto color) {
+        self.backgroundColor = [NSColor colorWithRed:(color.red() / 255.0)
```

Comment:
Maybe a little bit to late but this could use the `ns_color_to_gfx_color` function